src/menu-helpers.js

```diff
@@ -93,6 +97,11 @@ function cloneMenuItem(item) {
     'beforeGroupContaining',
     'afterGroupContaining'
   );
+  if (item.id === null || item.id === undefined) {
+    // item.id is internally used by the function defined in this file, because `atom-i18n` localize item.label.
+    // cloneMenuItem() create menus from template (menu_{darwin, linux, win32}.cson), then `atom-i18n` can localize menu.
```

Comment:
```suggestion
```

Reply:
I see. I delete the lines, according to your advice